scripts/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
 rm -rf dist
+npm run tsc -- --project tsconfig.build.json
+# Rename ts declarations output 'src' -> 'types'
+mv dist/src dist/types
```

Comment:
Yeah, I was battling this one for a while and finally just settled and decided to do this copy. I'll look into using the environment vars though because that would be much more straightforward 👍